src/rules.js

```diff
@@ -1310,7 +1310,14 @@ module.exports = [
         "allowParamToValue": true,
         "styles": {
             "font-size": "$0"
-        }
+        },
+        "arguments": [{
+            "xs": "0.7rem",
+            "s": "0.8rem",
```

Comment:
you guys can changes those values to ones that would make more sense in most apps